Description: Don't link against the vtk-java modules
 I could be utterly wrong, but AAFAIK Java modules are not
 shared libraries in the sense  that one would link against
 them at compile time. Instread they are loaded dynamically
 during run-time by the java run-time engine.
Author: Gert Wollny  <gw.fossdev@gmail.com>

--- a/Utilities/VTK/CMakeLists.txt
+++ b/Utilities/VTK/CMakeLists.txt
@@ -670,9 +670,9 @@
     endif()
     target_link_libraries(${VTKGDCM_NAME}Java ${VTKGDCM_NAME})
     set_property(TARGET ${VTKGDCM_NAME}Java PROPERTY NO_SONAME 1)
-    foreach(c ${vtkgdcm_LIBS})
-      target_link_libraries(${VTKGDCM_NAME}Java ${c}Java)
-    endforeach()
+#    foreach(c ${vtkgdcm_LIBS})
+#      target_link_libraries(${VTKGDCM_NAME}Java ${c}Java)
+#    endforeach()
     # Create the jar file:
     # I am pretty sure this *.java thingy will bite me one day, when someone will try
     # to recompile from an existing build tree with invalid generated *.java file...
